services/synthesis_service.py

```
@@ -0,0 +1,224 @@
+            
+            logger.info(f"Generated answer in {generation_ms}ms, {completion_tokens} tokens, {len(used_citation_ids)} citations")
+            
+            return SynthesisResult(
+                answer_markdown=answer,
+                used_citation_ids=used_citation_ids,
+                prompt_tokens=prompt_tokens,
+                completion_tokens=completion_tokens,
+                generation_ms=generation_ms
+            )
+            
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)